test/language/expressions/unary-minus/bigint-non-primitive.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Unary minus for BigInt object wrappers
+esid: sec-unary-minus-operator-runtime-semantics-evaluation
+info: |
+  Runtime Semantics: Evaluation
+  UnaryExpression : - UnaryExpression
+
+  1. Let expr be the result of evaluating UnaryExpression.
+  2. Let oldValue be ? ToNumeric(? GetValue(expr)).
+  3. Let T be Type(oldValue).
+  4. Return ? T::unaryMinus(oldValue).
+
+features: [BigInt, Symbol.toPrimitive]
+---*/
+
+assert.sameValue(-Object(1n), -1n, "-Object(1n) === -1n");
+assert.notSameValue(-Object(1n), 1n, "-Object(1n) !== 1n");
```

Comment:
you already assert that `-Object(1n)` evaluates `-1n`. You don't need to assert any of the `nonSameValue`. The tests for `===` can capture the results for the primitives.

The same apply to the other `notSameValue` cases here.